GliaWidgets/Sources/View/Chat/GVA/GvaPersistentButtonOptionView.swift

```diff
@@ -1,6 +1,8 @@
 import UIKit
 
 class GvaPersistentButtonOptionView: BaseView {
+    static let height: CGFloat = 42
```

Comment:
what is the benefit of using static height here?

Reply:
I use it for calculating GvaGalleryCardView height
https://github.com/salemove/ios-sdk-widgets/pull/706/files#diff-1d969513101fdb7676b4188db6ec31945a0bb05f78719b387163d3812ce5225dR61